packages/devtools/src/server-rpc/assets.ts

```diff
@@ -12,8 +12,8 @@ export function setupAssetsRPC({ nuxt, ensureDevAuthToken, refresh, options }: N
   let cache: AssetInfo[] | null = null
 
   const extensions = options.assets?.uploadExtensions || defaultAllowedExtensions
-
   const publicDir = resolve(nuxt.options.srcDir, nuxt.options.dir.public)
+  const layerDirs = [publicDir, ...nuxt.options._layers.map(layer => resolve(layer.cwd, layer.config?.public ?? 'public'))]
```

Comment:
`public` is not presented in the type - can you double check with this?

Reply:
I'll go with hard-coded one first, feel free to update it later if there is a better way to do so

Reply:
@antfu okay, thanks